inst/@sym/sym.m

```diff
@@ -158,212 +158,224 @@
 %% Author: Colin B. Macdonald
 %% Keywords: symbolic, symbols, CAS
 
-function s = sym(x, varargin)
-
-  if (nargin == 0)
-    s = sym(0);
-    return
+classdef sym < handle
+  properties
```

Comment:
Maybe make some or all of these properties `Access=Protected` or `Private`? Some are used outside of the class but many are only used directly inside class methods.